### PR TITLE
Add jest for testing

### LINKED ISSUE #14: Tests
We should have some of these…

### PR DESCRIPTION
**Pull request by spinningarrow**
This will close #14 once it's done.

### CODE REVIEW

**Comment by spinningarrow** [comment]
I'll start with some actual tests once #15 is closed.

**Comment by xevenheaven** [comment]
No problemo!

File: src/components/KudosList/spec.jest.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		let element = TestUtils.renderIntoDocument(<KudosList/>);
+		let div = TestUtils.findRenderedDOMComponentWithTag(element, 'div');
+
+		// Verify that the div is empty
+		expect(div.getDOMNode().textContent).toEqual('');
+	});
+
+	// doesn't show all kudos - only the current user's
+	// shows kudos in reverse chrono order
+	// shows a relative date?
```
Comment (xevenheaven) [inline src/components/KudosList/spec.jest.js:25]: Just being anal. Capitalise the first word? :tongue:

Comment (spinningarrow) [inline src/components/KudosList/spec.jest.js:25]: Haha, yes. These are just placeholders! I found an unpushed commit on my computer which I must've done ages ago :P

**Comment by spinningarrow** [comment]
Nice article on TDD: http://blog.testdouble.com/posts/2014-01-25-the-failures-of-intro-to-tdd.html

**Pull request closed without merging**